### PR TITLE
Fix teknoparrot titles and add gun + wheel games

### PR DESCRIPTION
**Pull request by Tartifless**
Uppercase is necessary, else ES does not find the games

### CODE REVIEW

**Comment by Tovarichtch** [comment]
Similar issue with #1732 about uppercases. As long as it doesn't touch generated `arcaderoms.xml` file, it's fine.

PR looks good. However, this is optional but futureproof: do you know the max wheel rotation for each of these systems? Or TeknoParrot doesn't care?

File: resources/gamesdb.xml [hunk 1]
```diff
@@ -19,6 +19,9 @@
     <game name="2spicy">
       <gun/>
     </game>
+    <game name="2Spicy">
+      <gun/>
+    </game>
```
Comment (Tovarichtch) [inline resources/gamesdb.xml:24]: Duplicate?

Comment (Tartifless) [reply resources/gamesdb.xml:24]: No, uppercase

**Comment by Tartifless** [comment]
TP does not care about max wheel rotation, neither does windows generally

File: resources/gamesdb.xml (hunk 1 above)
Comment (Tovarichtch) [reply resources/gamesdb.xml:24]: I see two times the game name. One uppercase, one lowercase. Game is both names?

**Comment by Tartifless** [comment]
2spicy lowercase already existed, for mame

**Comment by Tovarichtch** [comment]
All good then. OK to merge.

**Pull request merged**